README.md

```diff
@@ -13,20 +13,29 @@ Maintain SKU specific package content such as templates and other static files.
 
 * Build SKU package
 
+  * Create sku pack with SKU build-package tool and on the root folder run:
+
   ```
-  $ ./build-package.bash <sku_pack_directory> <subname>
+  $ ./build-package.bash <sku_pack_name> <subname>
+  Note: for below command <sku_pack_name> must be exactly the same with one of the node names listed on the root folder of on-skupack,
+  e.g. it can be quanta-d51-1u, quanta-t41,dell-r630, etc.<subname> can be any name you like. A {sku name}.tar.gz will be created in
+  tarball folder of the same directory.
   ```
   
   * Example
   
   ```
   $ ./build-package.bash quanta-t41 master
+  Note: After running this command, quanta-t41_master.tar.gz could be found at the tarball folder under the on-skupack root folder.
   ```
   
 * Register a new SKU with the package
 
   ```
   $ curl -X POST -F file=@tarballs/sku_pack_directory_subname.tar.gz localhost:8080/api/current/skus/pack
+  Note: The above command will return a SKU ID which will be used to post the Workflow against the node. If an error like “Duplicate entry exists”
```

Comment:
7.	The workflow to flash a new BMC image to a Quanta node needs to be POST’ed
POST Workflow
Sorry for any confusion, but it will, for example, when upgrading Quanta BMC, the command and usage is as following:
POST /api/2.0/nodes/:id/workflows?name=Graph.Flash.Quanta.Bmc::sku.ID
curl -X POST <server>/api/2.0/nodes/<nodeid>/workflows?name=Graph.Flash.Quanta.Bmc::sku.ID